controller-server/src/test/java/com/yahoo/vespa/hosted/controller/restapi/application/responses/search-deployments-single.json

```diff
@@ -0,0 +1,12 @@
+{
+  "deployments": [
+    {
+      "tenant": "tenant1",
+      "application": "application1",
+      "instance": "instance1",
+      "environment": "prod",
+      "region": "us-central-1",
```

Comment:
Maybe we should include the cluster name here as well.

Reply:
Yes. 

Also, tenant and application are always the same, but keeping all the attributes together may be fine just for simplicity anyway.